packages/extension-base/src/background/KoniTypes.ts

```diff
@@ -39,6 +39,7 @@ import { SignerResult } from '@polkadot/types/types/extrinsic';
 import { HexString } from '@polkadot/util/types';
 
 import { EarningImpactResult } from '../services/earning-service/handlers/native-staking/dtao';
+import { RequestAddSubstrateProxy, RequestGetSubstrateProxyAccounts, RequestRemoveSubstrateProxy, SubstrateProxyAccounts } from '../types/substrate-proxy';
```

Comment:
Rename SubstrateProxyAccounts to better name;

substrate-proxy -> substrate-proxy-account